openedx/features/pakx/lms/pakx_admin_app/serializers.py

```diff
@@ -31,12 +28,38 @@ def get_enrollment_date(obj):
         return obj.created.strftime('%Y-%m-%d')
 
     def get_progress(self, obj):
-        return get_course_progress_percentage(self.context['request'], text_type(obj.course.id))
+        # todo: refactor this by refactoring courseprogressstats relation with user/organization
+        course_stats = [c for c in obj.user.courseprogressstats_set.all() if c.course_id == obj.course_id]
+        return course_stats[0].progress if course_stats else None
```

Comment:
shouldn't we filter against user & course ID as we know there will be only 1 record for each enrolment?
and if no record is found return 0?